src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
-  external json: t => 'a => unit = "" [@@bs.send];
+  external sendString : t => string => unit = "send" [@@bs.send];
+  external sendObject : t => Js.t 'a => unit = "send" [@@bs.send];
```

Comment:
Cool, I wasn't aware of that syntax. Is that a Js.t type wrapping an arbitrary record? I'd like to avoid tracking master if we can, but if you think it's appropriate here I'll make that change.